### DISCUSSION TITLE
Examples in use

### DISCUSSION THREAD

**Discussion by thompson318** [SciKit-Surgery/sustainable-pkg-stats#22] [General]
https://thompson318.github.io/dashboard_for_scikit-surgery/

https://thompson318.github.io/dashboard_for_scikits/

https://thompson318.github.io/dashboard_for_magpi/

https://thompson318.github.io/dashboard_for_microbit/

https://thompson318.github.io/dashboard_for_thompson318/

**Comment by niksirbi** [comment]
https://neuroinformatics-unit.github.io/dashboard_for_neuroinformatics-unit/

**Comment by mxochicale** [comment]
https://mxochicale.github.io/dashboard_for_testing/
